kobo/settings/base.py

```diff
@@ -235,6 +235,9 @@
     'MFA_LOCALIZED_HELP_TEXT': (
         json.dumps({
             'default': (
+                # It's terrible, but if you change this, you must also change
+                # `MFA_DEFAULT_HELP_TEXT` in `static_lists.py` so that
+                # translators receive the new string
```

Comment:
Perhaps a more sane approach would be to have superusers leave the default value empty when they do not want to override it. However, doing that would take away our ability to use the default string to show an instructional example. Copying and pasting the string seemed like the lesser evil :upside_down_face: